grodecoder.py

```diff
@@ -862,6 +862,70 @@ def find_ion_solvant(
     return (universe, counts)
 
 
+def count_remove_ion_solvant(
```

Comment:
This function looks redundant with `find_ion_solvant()`
Could you have a look @KarinDuong ?